samples/client/petstore/powershell_test/src/IO.Swagger/Model/New-Order.ps1

```diff
@@ -0,0 +1,36 @@
+function New-Order {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[Int64]]
+        ${ id },
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[Int64]]
+        ${ petId },
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[Int32]]
+        ${ quantity },
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[JavaTimeLocalDateTime]]
```

Comment:
Wrong type mapping slipped in. This should probably be `[System.Nullable[System.DateTime]]`.